playbooks/roles/common-ses/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Include global vars
+  include_vars: "{{ playbook_dir }}/../vars/common-vars.yml"
+
+- name: Create pools in SES for socok8s usage
+  become: yes
+  include_role:
+    name: ses
+    tasks_from: openstack_config
```

Comment:
is this task idempotent? Since it doesn't look like we added code to delete the roles when uninstall, would redeploy still work? I admit that I haven't checked the openstack_config code.

Reply:
Yes, it is as I have ran this multiple times. create pool does fail if it already exists but ansible ignores that with failed_when: false logic.